src/lib/components/globalhealthbar/GlobalHealthBar.component.js

```diff
@@ -0,0 +1,139 @@
+//@flow
+import React from 'react';
+import VegaChart from '../vegachart/VegaChart.component.js';
+
+type Props = {
```

Comment:
exporting prop type could be useful to ease app integration 
```suggestion
export type GlobalHealthProps = {
```